be/src/connector/hive_connector.cpp

```diff
@@ -406,6 +407,36 @@ HdfsScanner* HiveDataSource::_create_hudi_jni_scanner() {
     return scanner;
 }
 
+HdfsScanner* HiveDataSource::_create_paimon_jni_scanner() {
+    const auto& scan_range = _scan_range;
+    const auto* paimon_table = dynamic_cast<const PaimonTableDescriptor*>(_hive_table);
+
+    std::string required_fields;
```

Comment:
please conside to support nested type. you can use `build_nested_fields` this function to build nested type fields.